app/views/author/submissions/acknowledge.html.erb

```diff
@@ -0,0 +1,26 @@
+<div class="row">
+    <h1><%= I18n.t("graduate.submission.acknowledge.title") %></h1>
+    <div class="col-xs-12">
+      <p class="mt-3"><%= I18n.t("graduate.submission.acknowledge.description") %> </p>
+      <%= simple_form_for @ack, url: author_submission_acknowledge_update_path, method: :patch do |form| %>  
+        <div class="form-group actions col-xs-6">          
+          <%= form.label :sig_1, I18n.t("graduate.submission.acknowledge.point1", degree_type: @degree_type, thesis_guide: I18n.t("graduate.partner.thesis_guide")).html_safe %>
+          <%= form.input :sig_1, label: false, placeholder: I18n.t("graduate.submission.acknowledge.initial").html_safe %>
+          <%= form.label :sig_2, I18n.t("graduate.submission.acknowledge.point2", degree_type: @degree_type.capitalize)  %>
+          <%= form.input :sig_2, label: false, placeholder: I18n.t("graduate.submission.acknowledge.initial") %>
+          <%= form.label :sig_3, I18n.t("graduate.submission.acknowledge.point3", degree_type: @degree_type) %>
+          <%= form.input :sig_3, label: false, placeholder: I18n.t("graduate.submission.acknowledge.initial") %>
+          <%= form.label :sig_4, I18n.t("graduate.submission.acknowledge.point4")  %>
+          <%= form.input :sig_4, label: false, placeholder: I18n.t("graduate.submission.acknowledge.initial") %>
+          <%= form.label :sig_5, I18n.t("graduate.submission.acknowledge.point5", degree_type: @degree_type)  %>
+          <%= form.input :sig_5, label: false, placeholder: I18n.t("graduate.submission.acknowledge.initial") %>
+          <%= form.label :sig_6, I18n.t("graduate.submission.acknowledge.point6")  %>
+          <%= form.input :sig_6, label: false, placeholder: I18n.t("graduate.submission.acknowledge.initial") %>
+          <%= form.label :sig_7, I18n.t("graduate.submission.acknowledge.point7") %>
+          <%= form.input :sig_7, label: false, placeholder: I18n.t("graduate.submission.acknowledge.initial") %>
+          <%= form.button :submit, class: "btn-primary", value: "Submit", disabled: @is_good %>
```

Comment:
Just curious about the `disabled: @is_good`.  Does that need to be there?  I might be missing something.

Reply:
That's my mistake...it's leftover from an earlier experiment I was trying. I'll remove it.